src/Stores/BasicCheckpointStore.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Plank\Checkpoint\Stores;
+
+use Plank\Checkpoint\Contracts\CheckpointStore;
+use Plank\Checkpoint\Models\Checkpoint;
+
+class BasicCheckpointStore implements CheckpointStore
+{
+    /**
+     * A variable to store the active Checkpoint
+     *
+     * @var null|Checkpoint
+     */
+    protected ?Checkpoint $checkpoint = null;
```

Comment:
Fix for php 7.

Reply:
good catch, I haven't spotted all of them.